faf-commons-api/src/main/java/com/faforever/commons/api/dto/LeagueSeasonDivisionSubdivision.java

```diff
@@ -0,0 +1,24 @@
+package com.faforever.commons.api.dto;
+
+import com.github.jasminb.jsonapi.annotations.Relationship;
+import com.github.jasminb.jsonapi.annotations.Type;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@Data
+@ToString(onlyExplicitlyIncluded = true, callSuper = true)
+@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
+@Type("leagueSeasonDivisionSubdivision")
+public class LeagueSeasonDivisionSubdivision extends AbstractEntity {
```

Comment:
technical name?

Reply:
the api doesn't provide a technical name on both of these